### PR TITLE
Re-skeletonize

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request re-skeletonizes this repository.  It also adds support for all the platforms we normally support, although that followed almost automatically from the re-skeletonization.

## 💭 Motivation and context ##

This repository needs to be re-skeletonized since it was based on an ancient version of a skeleton that lacked even a Lineage configuration.  It also needs to support Debian Buster, Bullseye, etc. so that it can support the plan to move CyHy off of Debian Stretch.

## 🧪 Testing ##

Automated testing passes.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Looks great, thanks for modernizing this one! 💎

File: molecule/default/prepare.yml [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+  ansible.builtin.import_playbook: python.yml
+
 # rsyslog is installed anywhere we would be applying this Ansible role
-- name: Prepare
+- name: Install rsyslog
   hosts: all
   tasks:
     # This is required temporarily, because of
     # https://github.com/ansible/ansible/issues/56583
     - name: Temporary workaround for ansible/ansible#56583
```
Comment (mcdonnnj) [inline molecule/default/prepare.yml]: Do we still need this now that the linked issue is closed? Per https://github.com/cisagov/ansible-role-banner/commit/3cddad3734b6924d0713b81b4a56c557abf70e35 it was resolved and subsequently removed from that repository.

File: molecule/default/requirements.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
 ---
 - src: https://github.com/dhs-ncats/ansible-role-logrotate
```
Comment (mcdonnnj) [inline molecule/default/requirements.yml]:
```suggestion
- src: https://github.com/cisagov/ansible-role-logrotate
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Thanks for helping drag another repo back into the light. I have a question and a suggestion for your consideration. I also wanted to bring up that the dependency [cisagov/ansible-role-logrotate](https://github.com/cisagov/ansible-role-logrotate) needs to be updated the same way as this repo.

File: molecule/default/prepare.yml (hunk 1 above)
Comment (jsf9k) [reply molecule/default/prepare.yml]: Good point.  I meant to follow up on that but forgot.  Please see commit 15e759d.

**Review by mcdonnnj — APPROVED** [review]
:shipit:

**Comment by jsf9k** [comment]
> I also wanted to bring up that the dependency [cisagov/ansible-role-logrotate](https://github.com/cisagov/ansible-role-logrotate) needs to be updated the same way as this repo.

@mcdonnnj - Please see cisagov/ansible-role-logrotate#8.

**Pull request merged**